Tests/ScipioKitTests/RunnerTests.swift

```diff
@@ -178,6 +178,43 @@ final class RunnerTests: XCTestCase {
         }
     }
 
+    func testWithPlatformMatrix() async throws {
+        let runner = Runner(
+            mode: .prepareDependencies,
+            options: .init(
+                buildConfiguration: .release,
+                isSimulatorSupported: true,
+                isDebugSymbolsEmbedded: false,
+                frameworkType: .dynamic,
+                cacheMode: .project,
+                platformMatrix: ["ScipioTesting": [.iOS, .watchOS]],
+                verbose: false)
```

Comment:
Missing test code for `sdks`: [`OrderedSet<SDK>`](https://github.com/giginet/Scipio/pull/19/files#diff-a520d2c4e21036a04374c4e7db9643e6ef28489b955fa9d14b823b69044fd3e4R10)

Reply:
These are another structs